src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -462,7 +462,7 @@ const cy: typeof en = (
           userCase.state === State.AwaitingClarification ? 'Beth sydd angen i chi ei wneud' : 'Diweddariad diweddaraf'
         }`,
   finalOrderComplete: {
-    line1: `Mae’r llys wedi caniatáu gorchymyn terfynol ichi. Mae eich ${isDivorce ? 'priodas' : 'partneriaeth sifil'} 
+    line1: `Mae’r llys wedi caniatáu gorchymyn terfynol ichi. Mae eich ${isDivorce ? 'priodas' : 'partneriaeth sifil'}
```

Comment:
is the whitespace not needed there?

Reply:
I have no recollection of changing this - either way it's not a required change for this ticket.  Reverted.